zipkin/src/main/java/com/linecorp/armeria/common/tracing/RequestContextCurrentTraceContext.java

```diff
@@ -41,10 +44,16 @@
  */
 public final class RequestContextCurrentTraceContext extends CurrentTraceContext {
 
+    /**
+     * Use this singleton when building a {@linkplain brave.Tracing} instance for use with
+     * {@linkplain HttpTracingService} or {@linkplain HttpTracingClient}.
```

Comment:
Uh, these are type names, so we should use `@link` rather than `@linkplain` here.

Reply:
hah we were just discussing this :P